service/src/main/java/bio/terra/cbas/common/MetricsUtil.java

```diff
@@ -161,6 +173,10 @@ public static void registerAllViews() {
                     25000.0,
                     50000.0)));
 
+    Aggregation recordsPerRequestDistribution =
+        Aggregation.Distribution.create(
+            BucketBoundaries.create(Arrays.asList(0.0, 10.0, 20.0, 30.0, 40.0, 50.0, 100.0)));
```

Comment:
I was going to say "we can probably drop the 0"... but maybe it's worth leaving, just in case we have some bug that means we're submitting 0 workflows to CBAS from time to time...